lib/checks/shared/none-empty-text.js

```diff
@@ -0,0 +1,2 @@
+var text = node.textContent;
```

Comment:
Should this check consider other content types such as images / video / frames, etc? If so, you may want to use `has-visible-text` instead.